python/ray/tune/tests/test_tuner.py

```diff
@@ -522,18 +522,18 @@ def to_dict(self) -> dict:
 
 
 def test_tuner_restore_classmethod():
-    tuner = Tuner("PPO")
+    tuner = Tuner(lambda x: None)
```

Comment:
Should we use `MyTrainableClass` everywhere?

Reply:
I think this one doesn't actually run the method so I didn't use it. I'm only using `MyTrainableClass` where checkpointing / restoration / stopping criterion is needed.